.travis.yml

```diff
@@ -5,8 +5,9 @@ env:
     - TOX_ENV=py35 SERVER=development
 before_install:
     - make install-dev
-    - make runkinto &
     - if [[ $SERVER == development ]]; then pip install https://github.com/Kinto/kinto/tarball/master; fi
+    - make runkinto &
+    - sleep 5   # to give kinto migrate a chance to finish
```

Comment:
what about running make migrate explicitely before make runkinto?

Reply:
It isn't really just `make migrate`; Kinto startup itself takes a couple seconds.